src/raps/actions/swap.ts

```diff
@@ -68,11 +66,8 @@ export const estimateSwapGasLimit = async ({
   }
 
   const { sellTokenAddress, buyTokenAddress } = quote;
```

Comment:
these destructured arguments are no longer used `sellTokenAddress` and `buyTokenAddress`